packages/headless/src/app/reducers.ts

```diff
@@ -109,3 +110,4 @@ export const excerptLength = excerptLengthReducer;
 
 export const insightConfiguration = insightConfigurationReducer;
 export const insightInterface = insightInterfaceReducer;
+export const caseContext = caseContextReducer;
```

Comment:
The exported constant must be named `insightCaseContext` otherwise the context information does not make it to the search request.

```suggestion
export const insightCaseContext = caseContextReducer;
```